UI/Web/src/app/_services/gestures.service.ts

```diff
@@ -0,0 +1,23 @@
+let swipeCoord = [0, 0];
+let swipeTime = new Date().getTime();
+
+export function swipe(e: TouchEvent, when: string): string | undefined {
+
+    const coord: [number, number] = [e.changedTouches[0].clientX, e.changedTouches[0].clientY];
```

Comment:
This needs index check to ensure there is at least 1 event (even if always there, better to write safe code than assume)

Reply:
Added guard clause check for e.changedTouches.length < 1